docusaurus/sidebars.js

```diff
@@ -79,7 +79,7 @@ const sidebars = {
         "how_to_guides/using_llms",
         "how_to_guides/enable_streaming",
         "how_to_guides/generate_structured_data",
-        "how_to_guides/custom_validators",
+        "how_to_guides/custom_validator",
```

Comment:
i think the current plural guide is the latest. is there info missing from it from the singular version?